content/ai/concepts/generative-ai/generative-ai.md

```diff
@@ -0,0 +1,126 @@
+
+With **LLMs** and a technology called **diffusion** AI systems can now match or exceed other skills we often associate with intelligence such as:
+
+-Understanding and creating new text content.
+-Writing never before created stories and poems.
+-Analyzing content for tone and emotions.
+-Creating unique images and video.
+
+### Is it art?
+
+In 2022 Jason Allen’s AI generated work, Théâtre D’opéra Spatial, won first place in the digital category at the Colorado State Fair. The artist did indicate that he used the **GenAI** platform **Midjourney**. It was later discovered that the judges weren't familiar with the tool at the time of judging.
+
+The question does **GenAI** create art is now being asked. Parallels are being drawn. When photography was introduced, only painting had ever been considered art. Photography wasn't considered art and the same discussion took place. Now, few people would argue that photography can be art.
+
+Another argument is that since there is no advanced skill required from the operator. Some consider are the application of skills developed over many years. However, modern art has been about concept and technique - not necessarily skill.
+
+An additional argument is that the human element is not present in **GenAI** created works. The counter argument is that the creation is still driven by a person who has emotion, creativity, and intent.
+
+## Applications
+
+Generative AI is a combination of several technologies and has many applications. Effective LLMs are used to understand a natural language prompt and a technologies such as **Diffusion models** in the case of image generation.
+
+- Text and Writing
+- Images
+- Music
+- Code
+
+### Other areas
+
+While **GenAI** is generally discussed in the context of Text, Images, Music, and Coding there are many other active areas. Some of these include.
+
+- Voice such as high fidelity text to speech
+- Scientific Discovery - generating hypothesis, accelerating discovery
+- Chemistry - molecule and drug discovery
+- Medical imaging
+- 3D Models - text to space creation
+- Video - movies, animations, transition between real and generated
+- Education - personalizing, customizing lessons, tutoring
+- Many others!
+
+## Impact and Possibilities
+
+### Legality
+
+The legal aspects of Generative AI is not yet defined. Class action lawsuits due to training data showing up infringing on the rights of artists.
+
+Freedom of expression, in the US fair use allows limited use of copyrighted material without permission of the copyright holder.
+
+Such as commentary, search engines, news reporting, research so copyright law not to rigid. Transformative which is defined as a new expression, meaning, or message to the work.
```

Comment:
```suggestion
Uses are allowed, such as commentary, search engines, news reporting, research and other transformative work. Transformative is defined as bringing a new expression, meaning, or message to the work. It can be argued that GenAI works can be transformative.
```